### DISCUSSION TITLE
Consider moving the examples out of the Blackjax repo

### DISCUSSION THREAD

**Discussion by rlouf** [blackjax-devs/blackjax#423] [General]
I several main advantages:
- Reduce CI time in `blackjax`, and allow for longer notebook runs.
- Independent development
- Self-contained reference on sampling methods

This example repo would render the notebook in CI and display them with Executable Books. We could expand the repo with tutorials that are algorithm-specific, and add comparisons between algorithms on similar models. We could probably improve the CI to only run modified notebook, thus allow for longer runs.

It would eventually read as a nice book on sampling methods.

**Comment by junpenglao** [comment]
🤖 *This reply was drafted with Claude and reviewed by me (@junpenglao).*

Done — the examples now live in the standalone [sampling-book](https://github.com/blackjax-devs/sampling-book), rendered separately from the core library CI. Closing.
